woocommerce/woocommerce
# Don't render aria described by if the arg is not passed in add-to-cart.php

### Changes proposed in this Pull Request:

If a theme is using `add-to-cart.php` but not passing the new modified arg name `aria-describedby_text` it causes potentially a lot of PHP warnings.

e.g. in https://wordpress.org/support/topic/add-to-cart-php-errors-after-latest-update/ and https://wordpress.org/support/topic/version-9-0-lots-of-errors/


### How to test the changes in this Pull Request:

This issue arises in themes that use `wc_get_template_part` but in their `$args` they don't include `aria-describedby_text`

So if they omit it you'll see these warnings in logs: `Undefined array key "aria-describedby_text" in /var/www/html/wp-content/plugins/woocommerce/templates/loop/add-to-cart.php on line 40`

To reproduce this: 

1. Checkout `trunk`, install / activate storefront theme
2. comment out line 1387 of `plugins/woocommerce/includes/wc-template-functions.php` which is `'aria-describedby_text' => $product->add_to_cart_aria_describedby(),`
3. Load the f/e shop page
4. Check your logs for the warning mentioned above
5. Now checkout this branch and follow steps 1-3, and note the warning no longer shows.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Avoid PHP warnings if `add-to-cart.php` template does not pass `aria-describedby_text`

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
